Code/GraphMol/Depictor/DepictUtils.cpp

```diff
@@ -364,16 +364,11 @@ void getNbrAtomAndBondIds(unsigned int aid, const RDKit::ROMol *mol,
   unsigned int na = mol->getNumAtoms();
   URANGE_CHECK(aid, na);
 
-  RDKit::ROMol::ADJ_ITER nbrIdx, endNbrs;
-  boost::tie(nbrIdx, endNbrs) = mol->getAtomNeighbors(mol->getAtomWithIdx(aid));
-
-  unsigned int ai, bi;
-  while (nbrIdx != endNbrs) {
-    ai = (*nbrIdx);
-    bi = mol->getBondBetweenAtoms(aid, ai)->getIdx();
+  for (auto ai : boost::make_iterator_range(
+           mol->getAtomNeighbors(mol->getAtomWithIdx(aid)))) {
```

Comment:
This is just `mol->atomNeightors()`, is it?

Reply:
here we need the index, not the atom itself, so using `getAtomNeighbors()` makes more sense.